java/algorithm/pom.xml

```diff
@@ -20,6 +20,10 @@
   <name>Arrow Algorithms</name>
   <description>(Experimental/Contrib) A collection of algorithms for working with ValueVectors.</description>
 
+  <properties>
+    <spotless.version>2.42.0</spotless.version>
```

Comment:
This version does not work with Java 8

Reply:
IMO, it's OK for this to only be Java 11+. The only thing that needs to work right now on Java 8 is build and test, not necessarily static analysis.

Reply:
For people still using Java 8 to build Arrow, it means they may not notice a formatting issue (or apply a fix for it) unless they submit a pull request or use a different java version. Workable but not ideal

Reply:
Earlier I added a separate version for Java 8, but we settled to keep just one. @lidavidm should we add it back?

Reply:
Could we use spotless [2.30.0](https://www.atlassian.com/agile/project-management/epics-stories-themes) until the build uses a newer java version?

Reply:
I can definitely try that.